core/core.go

```diff
@@ -32,9 +32,9 @@ func NewCore(conf Configuration) (*Core, error) {
 	}, nil
 }
 
-func (c *Core) CreateOrganization(namespacedOrganization string) error {
+func (c *Core) CreateOrganisation(namespacedOrganisation string) error {
```

Comment:
Can you use "organization" as in American English? It is also used as "organization" in Lifter/Vamp.

Reply:
Ok